### PR TITLE
KOGITO-9057 Remove Streams for Apache Kafka content from Serverless Logic Web Tools

### PR DESCRIPTION
**Pull request by caponetto**
**JIRA:** https://issues.redhat.com/browse/KOGITO-9057

**Description:** Removing content related to Streams for Apache Kafka since the tab was removed in https://github.com/kiegroup/kie-tools/pull/1583

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [NA] The nav.adoc file has a link to this guide in the proper category
- [NA] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-deploy-projects.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 :description: {serverless_logic_web_tools_name} deploying your projects
 :keywords: kogito, workflow, serverless, editor, web, tools, settings, openshift, deploy, project
 
-You can deploy your {product_name} projects to an OpenShift instance using the OpenShift integration. The OpenShift integration allows you test your implementations in a live environment. 
+You can deploy your {product_name} projects to an OpenShift instance using the OpenShift integration. The OpenShift integration allows you test your implementations in a live environment.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-deploy-projects.adoc:7]:
```suggestion
You can deploy your {product_name} projects to an OpenShift instance using the OpenShift integration. The OpenShift integration allows you to test your implementations in a live environment.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-deploy-projects.adoc [hunk 2]
```diff
@@ -25,7 +25,7 @@ For more information, see xref:tooling/serverless-logic-web-tools/serverless-log
 . Create a project using an example application in link:{kogito_sw_examples_url}[GitHub].
 +
 --
-You can use the link:https://github.com/kiegroup/kogito-examples/tree/main/serverless-workflow-examples/serverless-workflow-greeting-quarkus[`serverless-workflow-greeting-quarkus`] example application, which does not contain external dependencies, and is easy to test and comprehend. 
+You can use the link:https://github.com/kiegroup/kogito-examples/tree/main/serverless-workflow-examples/serverless-workflow-greeting-quarkus[`serverless-workflow-greeting-quarkus`] example application, which does not contain external dependencies, and is easy to test and comprehend.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-deploy-projects.adoc:28]:
```suggestion
You can use the link:https://github.com/kiegroup/kogito-examples/tree/main/serverless-workflow-examples/serverless-workflow-greeting-quarkus[`serverless-workflow-greeting-quarkus`] example application, which does not contain external dependencies and is easy to test and comprehend.
```

**Comment by caponetto** [comment]
Thanks, @kaldesai! Changes are in!

**Comment by ricardozanini** [comment]
@caponetto can we merge this?

**Comment by caponetto** [comment]
> @caponetto can we merge this?

Yes please.

**Pull request merged**